### DISCUSSION TITLE
are possible to use single online sqlite database ?

### DISCUSSION THREAD

**Discussion by koncoweb** [SonicJs-Org/sonicjs#180] [Q&A]
we assume the scenario : sonicjs installed locally on client pcs / laptops , but it access single database from cloud for data unity ? is that possible ?

**Comment by lane711** [comment]
Hi, I haven't tried that but if you want to test it you should be able to update the path here in server.js line 360:

<img width="796" alt="Screenshot 2023-01-29 at 10 17 34 AM" src="https://user-images.githubusercontent.com/4350219/215347430-6d50b85a-490a-4b60-9006-0ed7d9409f64.png">

SonicJs uses TypeORM so you might want to Google that for additional info as well as I'm sure someone else has wanted to do this.
